configs/config.go

```diff
@@ -14,6 +14,9 @@ This is an initial implementation of Config. Please keep in mind we're still wor
 environment variables and general config data.
 */
 
+// to use with parans ldlfags
+var k1Version string
```

Comment:
is this var receiving the value from a environment variable?

Reply:
it seems to be this one: `export LDFLAGS="-X github.com/kubefirst/kubefirst/configs.k1Version=1.8.6"`

Reply:
A building flag mechanism.